templates/blog/article.html

```diff
@@ -0,0 +1,177 @@
+    "@id": "https://ubuntu.com/#article",
+    "@type": "Article",
+    "name": "{{ article.title.renderered|safe }}",
+    "headline": "{{ article.excerpt.raw }}",
+    "author": {
+       "@type": "Person",
+       "name": "{{ article.author.name }}"
+    },
+    "datePublished": "{{ article.date_gmt }}",
+    {% if article.image and article.image.source_url %}
+      "image": "{{ article.image.source_url }}",
+    {% endif %}
+    "url": "{{ request.url }}",
+    "publisher": {
+      "@type": "Organization",
+      "name": "Ubuntu"
+    }
+  }
+  </script>
+{% endblock %}
+
+{% block content %}
+<article>
+  <header class="p-strip--suru">
+    <div class="row">
+      <div class="col-8">
+        <h1>{{ article.title.rendered|safe }}</h1>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-8">
+        <div class="p-media-object">
+          {% if article.author %}
+            <img src="{% if article.author.id == 217 %}https://assets.ubuntu.com/v1/60d9b81e-picto-canonical.svg{% else %}{{ article.author.avatar_urls['96'] }}{% endif %}" class="p-media-object__image is-round" alt="">
+          {% endif %}
+
+          <div class="p-media-object__details">
+            {% if article.author %}
+              <h3 class="p-media-object__title">
+                <a href="/blog/author/{{ article.author.slug }}" title="More about {{ article.author.name }}">{{ article.author.name }}</a>
+              </h3>
+            {% endif %}
+            <p class="p-media-object__content">on {{ article.date }}</p>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-4">
+        <ul class="p-inline-list-icons u-no-padding--left u-no-margin--left">
+          <li class="p-inline-list__item">
+            Share on:
+          </li>
+
+          <li class="p-inline-list__item">
+            <a class="p-icon--facebook" title="Share on Facebook" href="https://www.facebook.com/sharer/sharer.php?u=https://www.ubuntu.com/blog/{{ article.slug }}">
+              Facebook
+            </a>
+          </li>
+
+          <li class="p-inline-list__item">
+            <a class="p-icon--twitter" title="Share on Twitter" href="https://twitter.com/share?text={{ article.title.rendered|safe|urlencode }}&amp;url=https://www.ubuntu.com/blog/{{ article.slug }}&amp;hashtags=ubuntu">
+              Twitter
+            </a>
+          </li>
+
+          <li class="p-inline-list__item">
+            <a class="p-icon--linkedin" title="Share on LinkedIn" href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https://www.ubuntu.com/blog/{{ article.slug }}&amp;title={{ article.title.rendered|safe|urlencode }}">
+            LinkedIn
+          </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    {% if tags %}
+      <div class="row">
+        <div class="col-10">
+          <p>
+            <strong>Tags:</strong>
+            {% for tag in tags %}
+              <a href="/blog/tag/{{ tag.slug }}">
```

Comment:
These links should have the [inverted styling](https://vanillaframework.io/docs/patterns/links#inverted).

Reply:
This is not applicable anymore, as design asked me to changed the strip to the light colored one we have on https://ubuntu.com/blog